### DISCUSSION TITLE
Ecommerce templates Products are linked to Stripe is not necessary

### DISCUSSION THREAD

**Discussion by qiwating** [payloadcms/payload#4951] [Feature Requests & Ideas]
Ecommerce templates Products are linked to Stripe is unnecessary,for a new website, you have to upload products to both side, your own site and stripe. 
why not give an choice, if the products exist in stripe , show price on stripe, if no on stripe, show the price directly already input on product create dashboard

**Comment by zubricks** [comment]
Hey @qiwating feel free to add this request to [this discussion](https://github.com/payloadcms/payload/discussions/4352). Appreciate your input!

**Comment by paulpopus** [comment]
We no longer sync data with Stripe

Closing this discussion in light of ecommerce plugin and template being released. We're going to capture discussions going forward in this RFC so we can better manage engagement on this topic: https://github.com/payloadcms/payload/discussions/14239

Thank you everyone!
